data/blog/new-website.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "Electron's New Internationalized Website"
+author: zeke
+date: '2017-11-13'
+---
+
+Electron has a new website at [electronjs.org]!  We've replaced 
+our static Jekyll site with a Node.js webserver, giving us flexibility to
+internationalize the site and paving the way for more exciting new features.
+
+## 🌍 Translations
+
+We've begun the process of internationalizing the website with the 
+goal of making Electron app development accessible to a global audience of 
+developers. We're using a localization platform called [Crowdin] that integrates 
+with  GitHub, opening and updating pull requests automatically as content is translated into different languages.
```

Comment:
Extra whitespace

Reply:
Thanks, @sethlu. Fixed the whitespace issues. Any feedback on the actual content?

Reply:
@zeke Content-wise I think it's well written! 👍 Looking forward to seeing it posted soon.